src/TestUtils/src/DeviceTests/AssertionExtensions.Android.cs

```diff
@@ -33,7 +33,8 @@ void OnLayout(object? sender = null, AView.LayoutChangeEventArgs? e = null)
 			{
 				var view = (AView)sender!;
 
-				view.LayoutChange -= OnLayout;
+				if (view.Handle != IntPtr.Zero)
```

Comment:
Just checking, is this change actually related?

Reply:
Not directly, but due to the async nature of this area of the tests it fails 34.8% of the time (I just made that up) So instead of a separate pr to fix that I am just adding it here. I think even this PR failed on that. And it is mostly safe due to it being test code.